src/menu/views/FilterMenu.js

```diff
@@ -25,6 +25,22 @@ const FilterMenu = MenuBuilder.extend({
       return this.g.columns.set("hidden", hidden);
     });
 
+    this.addNode("Hide non-polymorphic sites",(e) => {
+      var threshold = prompt("Enter identity threshold (in percent)", 100);
+      threshold = threshold / 100;
+      var maxLen = this.model.getMaxLength();
+      var hidden = [];
+      // TODO: cache this value
+      var conserv = this.g.stats.scale(this.g.stats.conservation());
```

Comment:
Don't over-complicate your life.

This operation will take less than a millisecond and requires interaction by the user.
[Conservation](https://github.com/wilzbach/stat.seqs) is already cached and the scaling is a simple map operation ;-)